### ISSUE TITLE
 apex.parallel.convert_syncbn_model has gradient overflow until the loss scale is reduced to zero

### ISSUE THREAD

**Issue by cuevhv** [NVIDIA/apex#953]
I am using multi GPU training. Previously I was using DDP from PyTorch with synchronized batch normalization. Now I'm trying to do the same using apex. The problem happens when I try to train the model using convert_syncbn_model. Basically, when I put this line, the loss scale gets re-adjusted until it becomes zero and the loss outputs NaNs. Bellow, I put a snippet of my code:

`torch.cuda.set_device(gpu)
net.cuda(gpu)
net = apex.parallel.convert_syncbn_model(net)
net, optimizer = apex.amp.initialize(net, optimizer, opt_level='O1')
net = DDP(net)`

` with apex.amp.scale_loss(loss, optimizer) as scaled_loss:
         scaled_loss.backward()`

Everything works fine when I do not use apex.parallel.convert_syncbn_model.

System:
Ubuntu: 16.04
Pytorch: 1.3.1 (the same happens with 1.4)
Apex: 0.1

**Comment by cuevhv** [comment]
I fixed this issue, it happens when I load the weights that came from a previous training that was in an overflow cycle which ended up in reducing the scale to zero (kind of corrupted weights).

**Comment by cuevhv** [comment]
After few epochs the problem persist. I've tested the same using the following system:
Ubuntu: 18.04
Pytorch: 1.6
Apex: 0.1
Cuda: 10.2
Unfortunately, the problem is still there, it seems that `parallel.convert_syncbn_model` has some problems with the gradient calculation.
